FBXLoader fails to load file

##### Description of the problem

I loaded the attached file by the example webgl_loader_fbx.html. And it failed with log on the console:

FBXLoader.js:109 Error: THREE.FBXLoader: Unknown property type 
    at BinaryParser.parseProperty (FBXLoader.js:3618)
    at BinaryParser.parseNode (FBXLoader.js:3371)
    at BinaryParser.parse (FBXLoader.js:3318)
    at FBXLoader.parse (FBXLoader.js:125)
    at Object.onLoad (FBXLoader.js:99)
    at XMLHttpRequest.<anonymous> (three.module.js:36859)

[JKXM.zip](https://github.com/mrdoob/three.js/files/4894876/JKXM.zip)

##### Three.js version

- [x] Dev
- [ ] r118
- [ ] ...

##### Browser

- [ ] All of them
- [x] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [ ] All of them
- [x] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)

--- a/examples/js/loaders/FBXLoader.js
+++ b/examples/js/loaders/FBXLoader.js
@@ -3263,7 +3263,11 @@ THREE.FBXLoader = ( function () {
 
 			var version = reader.getUint32();
 
-			console.log( 'THREE.FBXLoader: FBX binary version: ' + version );
+			if ( version < 6400 ) {
+
+				throw new Error( 'THREE.FBXLoader: FBX version not supported, FileVersion: ' + version );
+
+			}
 
 			var allNodes = new FBXTree();
 

--- a/examples/jsm/loaders/FBXLoader.js
+++ b/examples/jsm/loaders/FBXLoader.js
@@ -3309,7 +3309,11 @@ var FBXLoader = ( function () {
 
 			var version = reader.getUint32();
 
-			console.log( 'THREE.FBXLoader: FBX binary version: ' + version );
+			if ( version < 6400 ) {
+
+				throw new Error( 'THREE.FBXLoader: FBX version not supported, FileVersion: ' + version );
+
+			}
 
 			var allNodes = new FBXTree();
 

